types/react-simple-maps/package.json

```diff
@@ -26,6 +26,10 @@
         {
             "name": "Kouame Komenan ",
             "githubUsername": "komenank"
+        },
+        {
+            "name": "Pierce Morris",
+            "githubUsername": "piercejmorris"
```

Comment:
Are you intending on being an owner of this package, doing reviews, etc? This is more about DT infra than it is credit.

Reply:
Yes this is intentional, it seems the current owners are not really active, so would like to help get involved in further reviews/contributions